index.html

```diff
@@ -1795,13 +1774,33 @@ <h2>Resource Integrity</h2>
         </table>
 
         <p>
-Other [[MULTIBASE]] encoding values MAY be used, but interoperability is not
-guaranteed between implementations.
+Other Multibase encoding values MAY be used, but interoperability is not
+guaranteed between implementations using such.
```

Comment:
using such values.

Reply:
Also fixed in 3a076dd89efb196b94cea106c6720b7aa632b6e5. Thank you!